package.json

```diff
@@ -16,11 +16,13 @@
   "dependencies": {
     "@vitejs/plugin-react": "^2.2.0",
     "aos": "3.0.0-beta.6",
+    "dotenv": "^16.0.3",
```

Comment:
Why have you installed dotenv? 

To use environment variables in vite [see this](https://vitejs.dev/guide/env-and-mode.html)